test/integration/manifests/cns/ciliumconfigmap.yaml

```diff
@@ -22,6 +22,7 @@ data:
           "NodeSyncIntervalInSeconds": 30
       },
       "ChannelMode": "CRD",
+      "EnableSubnetScarcity": true,
```

Comment:
nothing to change here, but I haven't seen this mode in use before, could I get some context?

Reply:
since we scale in batches of IPs in swift, it is possible to have a lot of allocated but free IPs on the node - this mode will allow CNS to give back its extra IPs (effectively setting the batch size to 1) when the subnet is running out of IPs. DNC/RC will monitor the subnet utilization and push a CRD which CNS to signal CNS. 
If the subnet is not constrained, this flag is a no-op, but I'm enabling it here now to make sure that it doesn't break anything if it's accidentally rolled out before the DNC changes.